src/sentry/static/sentry/app/views/performance/vitalDetail/vitalPercents.tsx

```diff
@@ -26,13 +29,46 @@ export default function VitalPercents(props: Props) {
         return (
           <VitalStatus key={p.vitalState}>
             {p.vitalState === VitalState.POOR && (
-              <IconFire color={vitalStateColors[p.vitalState] as Color} />
+              <Tooltip
+                title={
+                  Array.isArray(props.vital)
+                    ? 'Poor'
+                    : tct('Poor: >[threshold]ms', {
+                        threshold: webVitalPoor[props.vital],
+                      })
+                }
+              >
+                <IconFire color={vitalStateColors[p.vitalState] as Color} />
+              </Tooltip>
             )}
             {p.vitalState === VitalState.MEH && (
-              <IconWarning color={vitalStateColors[p.vitalState] as Color} />
+              <Tooltip
+                title={
+                  Array.isArray(props.vital)
+                    ? 'Needs improvement'
+                    : tct('Needs improvement: >[threshold]ms', {
+                        threshold: webVitalMeh[props.vital],
+                      })
+                }
+              >
+                <IconWarning color={vitalStateColors[p.vitalState] as Color} />
+              </Tooltip>
             )}
             {p.vitalState === VitalState.GOOD && (
-              <IconCheckmark color={vitalStateColors[p.vitalState] as Color} isCircled />
+              <Tooltip
+                title={
+                  Array.isArray(props.vital)
+                    ? 'Good'
+                    : tct('Good: <[threshold]ms', {
```

Comment:
This tooltip doesn't explain much more than what the visuals already show. Can be elaborate a little more on what this means? For example (and feel free to change it up)

> 40% of first contentful paint is below 2400ms.

Reply:
I thought about that, but I'm not sure how much additional information it adds. The percentage is already displayed right next to the tooltip, and the name of the metric is in large print on the left side of the table. The main point of this tooltip IMO is to add the threshold information. Maybe that should just be rendered directly and not hidden in a tooltip though?

Reply:
had a chat with Adam, we agreed that always showing it adds unnecessary noise to the page. i'm not that happy with it either, but i think it's fine and not that important, so let's merge it on Tuesday if nobody has any better ideas over the weekend?